mycore-webtools/src/main/java/org/mycore/webtools/vue/MCRVueRootServlet.java

```diff
@@ -184,7 +245,17 @@ protected org.jdom2.Document getIndexDocument(URL indexResource, String absolute
      */
     protected Properties getProperties() {
         Properties properties = new Properties();
+
+        MCRConfiguration2.getOrThrow(VUE_PASSTHROUGH_PROPERTY_NAME, MCRConfiguration2::splitValue)
+                .map(k -> new AbstractMap.SimpleImmutableEntry<>(k, MCRConfiguration2.getString(k)) {})
+                .filter(e -> e.getValue().isPresent())
```

Comment:
It might be useful to include empty properties to make it more explicit and consistent.